libraries/AP_Parachute/AP_Parachute.h

```diff
@@ -104,6 +108,7 @@ class AP_Parachute {
     AP_Int16    _servo_off_pwm; // PWM value to move servo to when shutter is deactivated
     AP_Int16    _alt_min;       // min altitude the vehicle should have before parachute is released
     AP_Int16    _delay_ms;      // delay before chute release for motors to stop
+    AP_Float     _timeout;      // loss of control timeout value in seconds
```

Comment:
nitpick: if possible I like to keep the parameter definitions in the order they appear in the parameter list in the .cpp file or some other logical order.  It looks like someone put the _options parameter in the wrong place and also didn't add a comment .. let's put that aside for the moment and maybe put the new parameter below _critical_sink?